.travis.yml

```diff
@@ -20,107 +18,99 @@ services:
 
 env:
   matrix:
-    - BUILD_TYPE=linux      PYTHON_VERSION=27     CONF=debug      ARCH=x86_64
-    - BUILD_TYPE=linux      PYTHON_VERSION=34     CONF=debug      ARCH=x86_64
-    - BUILD_TYPE=linux      PYTHON_VERSION=35     CONF=debug      ARCH=x86_64
-    - BUILD_TYPE=linux      PYTHON_VERSION=36     CONF=debug      ARCH=x86_64
-    - BUILD_TYPE=linux      PYTHON_VERSION=pypy2  CONF=debug      ARCH=x86_64
-    - BUILD_TYPE=linux      PYTHON_VERSION=27     CONF=release    ARCH=x86_64
-    - BUILD_TYPE=linux      PYTHON_VERSION=34     CONF=release    ARCH=x86_64
-    - BUILD_TYPE=linux      PYTHON_VERSION=35     CONF=release    ARCH=x86_64
-    - BUILD_TYPE=linux      PYTHON_VERSION=36     CONF=release    ARCH=x86_64
-    - BUILD_TYPE=linux      PYTHON_VERSION=pypy2  CONF=release    ARCH=x86_64
-    - BUILD_TYPE=linux      PYTHON_VERSION=27     CONF=coverage   ARCH=x86_64
-    - BUILD_TYPE=manylinux  PYTHON_VERSION=27     DOCKER_IMAGE=keyvidev/keyvi-manylinux-builder
-    - BUILD_TYPE=manylinux  PYTHON_VERSION=34     DOCKER_IMAGE=keyvidev/keyvi-manylinux-builder
-    - BUILD_TYPE=manylinux  PYTHON_VERSION=35     DOCKER_IMAGE=keyvidev/keyvi-manylinux-builder
-    - BUILD_TYPE=manylinux  PYTHON_VERSION=36     DOCKER_IMAGE=keyvidev/keyvi-manylinux-builder
-    - BUILD_TYPE=sdist      PYTHON_VERSION=27
-    - BUILD_TYPE=doc
-    - BUILD_TYPE=rust
+    - BUILD_TYPE=linux      DOCKER_IMAGE=narekgharibyan/keyvi-ubuntu-builder      CONF=release
```

Comment:
I created a keyvi-ubuntu-builder  under keyvidev and added you as collaborator

Reply:
Thanks!